Devscord.DiscordFramework/Services/EmbedMessagesService.cs

```diff
@@ -37,8 +37,8 @@ private EmbedBuilder GetDefault()
             return new EmbedBuilder()
                 .WithThumbnailUrl(@"https://raw.githubusercontent.com/Devscord-Team/Watchman/master/avatar.png")
                 .WithFooter(new EmbedFooterBuilder()
-                .WithText(FOOTER_TEXT)
-                .WithIconUrl(@"https://raw.githubusercontent.com/Devscord-Team/Watchman/master/avatar.png"));
+                    .WithText(FOOTER_TEXT)
```

Comment:
czemu dorzuciłeś wcięcia?

Reply:
ponieważ to odnosi się do `new EmbedFooterBuilder()`, więc uznałem że lepiej będzie to wyglądać